planning/behavior_path_planner/autoware_behavior_path_planner_common/src/utils/utils.cpp

```diff
@@ -648,6 +656,26 @@ double getDistanceToEndOfLane(const Pose & current_pose, const lanelet::ConstLan
   return lanelet_length - arc_coordinates.length;
 }
 
+double getDistanceFromLastFitWidthToEnd(
+  const lanelet::ConstLanelet & lane, const double width_threshold)
+{
+  const auto center_line = lane.centerline3d().basicLineString();
+  double distance = 0.0;
+  if (center_line.size() <= 1) {
+    return distance;
+  }
+  auto it = center_line.rbegin() + 1;
+  for (; it < center_line.rend(); ++it) {
```

Comment:
Is it possible for the centerline to only contain 2 points ? In which case this code does not work well and interpolation is required to get an accurate distance.

Reply:
Modified implementation to check distance between points and interpolate if necessary